docs/source/tutorials/fitting-CBVs.rst

```diff
@@ -0,0 +1,181 @@
+
+Example 5: Fitting CBVs to Remove Long Term Trends
```

Comment:
GO style is to avoid capitalizing titles, i.e. "remove long term trends" instead of "Remove Long Term Trends".